static/js/components/Card.js

```diff
@@ -2,11 +2,12 @@
 import React from "react"
 
 type CardProps = {
-  children: React$Element<*>
+  children: React$Element<*>,
+  className?: string
 }
 
-const Card = ({ children }: CardProps) =>
-  <div className="card">
+const Card = ({ children, className }: CardProps) =>
+  <div className={className ? `card ${className}` : "card"}>
```

Comment:
I'm going to leave it as is just because what you have there leaves an extra space after `card ` which irks me a little bit